package.json

```diff
@@ -32,21 +33,24 @@
     "karma-mocha": "^0.2.0",
     "karma-mocha-reporter": "^1.1.0",
     "karma-sinon-chai": "^1.1.0",
+    "mkdirp": "0.5.1",
     "mocha": "^2.3.3",
     "mozilla-download": "^1.0.5",
+    "ncp": "2.0.0",
+    "open": "0.0.5",
     "replace": "^0.3.0",
+    "rimraf": "2.5.0",
     "semistandard": "^7.0.2",
-    "standard-reporter": "^1.0.5",
     "uglifyjs": "^2.4.10"
   },
   "scripts": {
     "start": "npm run dev",
     "dev": "npm run build && node ./scripts/budo",
     "browserify": "browserify ./src/index.js -s aframe-core",
-    "build": "mkdir -p build/ && npm run browserify -- --debug -o build/aframe-core.js",
-    "dist": "rm -rf dist/ && mkdir -p dist/ && npm run dist:js",
+    "build": "mkdirp build/ && npm run browserify -- --debug -o build/aframe-core.js",
```

Comment:
@cvan Just looked at (shelljs)[https://github.com/shelljs/shelljs]. How can you use directly from the command line/script directive without resorting to an external script? shelljs installs a ```shjs``` binary that takes a script file but I don't see any example that lets me do something like this: shjs "rm -rf dist/"